padertorch/summary/tbx_utils.py

```diff
@@ -210,14 +263,56 @@ def audio(signal, sampling_rate: int = 16000, batch_first=False,
     return signal, sampling_rate
 
 
+def matplotlib_figure_to_image(
```

Comment:
Can you motivate, why `matplotlib_figure_to_image` and not `matplotlib_figure`?
i.e. tensorboradX has also a figure option. They use much more code for the conversion: https://github.com/lanpa/tensorboardX/blob/8f50866d10ca064b7093c364985aa25d1dcd27a9/tensorboardX/utils.py#L2

Reply:
I didn't know that function. It could work equally well and additionally supports multiple figures and doesn't require setting the backend before calling this function